autodispose-android/build.gradle

```diff
@@ -49,6 +49,7 @@ dependencies {
   compile deps.rx.java
   compile deps.rx.android
   compile deps.support.annotations
+  compile "android.arch.lifecycle:extensions:1.0.0-alpha1"
```

Comment:
+1 for a separate module approach

Reply:
btw, you don't need extensions, you should only need lifecycle:common

Reply:
cool, I'll remove it

Reply:
c8ad885

Reply:
Actually I remember now why I added it, I need `LifecycleRegistry` for the `TestAndroidLifecycleScopeProvider` implementation. Added back in 1c06bef